doc/sphinx/Pacemaker_Remote/baremetal-tutorial.rst

```diff
@@ -264,37 +288,35 @@ the ``/usr/lib/ocf/resource.d/pacemaker/remote`` file that describes what option
 are available, but there is no actual **ocf:pacemaker:remote** resource agent
 script that performs any work.
 
-Define the remote node connection resource to our remote node,
+Before we integrate the remote node, we'll need to authorize it.
+.. code-block:: none
+    # pcs host auth remote1
+
+Now, define the remote node connection resource to our remote node,
 **remote1**, using the following command on any cluster node.
 
 .. code-block:: none
-
-    # pcs resource create remote1 ocf:pacemaker:remote
+    # pcs cluster node add-remote remote1
```

Comment:
These lines need to line up under the "c", otherwise they disappear (you can confirm the output with "make -C doc" and going to file:///*...path to checkout ...*/doc/sphinx/Pacemaker_Remote/_build/singlehtml/index.html)

Reply:
@kgaillot Thanks for the command. This is what happened when I tried to run it:

[gchin@gchin pacemaker]$ make -C doc
make: Entering directory '/home/gchin/pacemaker/doc'
cd .. && make  am--refresh
make[1]: Entering directory '/home/gchin/pacemaker'
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/gchin/pacemaker/missing aclocal-1.16 
 cd . && /bin/sh /home/gchin/pacemaker/missing automake-1.16 --foreign
configure.ac:43: error: required directory ./libltdl/config does not exist
configure.ac:102: error: required file 'libltdl/config/compile' not found
configure.ac:102:   'automake --add-missing' can install 'compile'
configure.ac:44: error: required file 'libltdl/config/config.guess' not found
configure.ac:44:   'automake --add-missing' can install 'config.guess'
configure.ac:44: error: required file 'libltdl/config/config.sub' not found
configure.ac:44:   'automake --add-missing' can install 'config.sub'
configure.ac:67: error: required file 'libltdl/config/install-sh' not found
configure.ac:67:   'automake --add-missing' can install 'install-sh'
configure.ac:130: error: required file 'libltdl/config/ltmain.sh' not found
configure.ac:67: error: required file 'libltdl/config/missing' not found
configure.ac:67:   'automake --add-missing' can install 'missing'
Makefile.am: error: required file 'libltdl/config/depcomp' not found
Makefile.am:   'automake --add-missing' can install 'depcomp'
cts/Makefile.am:33: error: required file 'libltdl/config/py-compile' not found
cts/Makefile.am:33:   'automake --add-missing' can install 'py-compile'
parallel-tests: error: required file 'libltdl/config/test-driver' not found
parallel-tests:   'automake --add-missing' can install 'test-driver'
make[1]: *** No rule to make target 'Makefile.in', needed by 'Makefile'.  Stop.
make[1]: Leaving directory '/home/gchin/pacemaker'
make: *** [Makefile:586: ../aclocal.m4] Error 2
make: Leaving directory '/home/gchin/pacemaker/doc'

Running the command suggested in the output didn't help:
[gchin@gchin pacemaker]$ automake --add-missing
configure.ac:43: error: required directory ./libltdl/config does not exist
configure.ac:102: error: installing 'libltdl/config/compile'; error while making link: No such file or directory
configure.ac:44: error: installing 'libltdl/config/config.guess'; error while making link: No such file or directory
configure.ac:44: error: installing 'libltdl/config/config.sub'; error while making link: No such file or directory
configure.ac:67: error: installing 'libltdl/config/install-sh'; error while making link: No such file or directory
configure.ac:130: error: required file 'libltdl/config/ltmain.sh' not found
configure.ac:67: error: installing 'libltdl/config/missing'; error while making link: No such file or directory
Makefile.am: error: installing 'libltdl/config/depcomp'; error while making link: No such file or directory
cts/Makefile.am:33: error: installing 'libltdl/config/py-compile'; error while making link: No such file or directory
parallel-tests: error: installing 'libltdl/config/test-driver'; error while making link: No such file or directory

I pushed a commit addressing all comments except this one.